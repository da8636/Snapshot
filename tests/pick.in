SET a 1 2 3
PICK a 2
BYE
