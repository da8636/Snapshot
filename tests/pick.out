ok

2

bye
